ush/global_cycle.sh

```diff
@@ -0,0 +1,347 @@
+FNALBC=${FNALBC:-${FIXorog}/${CASE}/sfc/${CASE}.mx${OCNRES}.snowfree_albedo.tileX.nc}
+FNVETC=${FNVETC:-${FIXorog}/${CASE}/sfc/${CASE}.mx${OCNRES}.vegetation_type.tileX.nc}
+FNSOTC=${FNSOTC:-${FIXorog}/${CASE}/sfc/${CASE}.mx${OCNRES}.soil_type.tileX.nc}
+FNABSC=${FNABSC:-${FIXorog}/${CASE}/sfc/${CASE}.mx${OCNRES}.maximum_snow_albedo.tileX.nc}
+FNVMNC=${FNVMNC:-${FIXorog}/${CASE}/sfc/${CASE}.mx${OCNRES}.vegetation_greenness.tileX.nc}
+FNVMXC=${FNVMXC:-${FIXorog}/${CASE}/sfc/${CASE}.mx${OCNRES}.vegetation_greenness.tileX.nc}
+FNSLPC=${FNSLPC:-${FIXorog}/${CASE}/sfc/${CASE}.mx${OCNRES}.slope_type.tileX.nc}
+FNMSKH=${FNMSKH:-${FIXgfs}/am/global_slmask.t1534.3072.1536.grb}
+NST_FILE=${NST_FILE:-"NULL"}
+FNTSFA=${FNTSFA:-${COMIN_OBS}/${PREINP}sstgrb${SUFINP}}
+FNACNA=${FNACNA:-${COMIN_OBS}/${PREINP}engicegrb${SUFINP}}
+FNSNOA=${FNSNOA:-${COMIN_OBS}/${PREINP}snogrb${SUFINP}}
+#  Other variables.
+PGMOUT=${PGMOUT:-${pgmout:-'out'}}
+PGMERR=${PGMERR:-${pgmerr:-'err'}}
+# Set defaults
+################################################################################
+#  Preprocessing
+
+ln -fs "${FNTSFC}" sstclm
+ln -fs "${FNSALC}" salclm
+
+# If the appropriate resolution fix file is not present, use the highest resolution available (T1534)
+[[ ! -f ${FNSMCC} ]] && FNSMCC="${FIXgfs}/am/global_soilmgldas.statsgo.t1534.3072.1536.grb"
+
+################################################################################
+#  Make surface analysis
+export PGM=${CYCLEXEC}
+export pgm=${PGM}
+
+iy=${PDY:0:4}
+im=${PDY:4:2}
+id=${PDY:6:2}
+ih=${cyc}
+
+export OMP_NUM_THREADS=${OMP_NUM_THREADS_CY:-${CYCLETHREAD:-1}}
+
+cat << EOF > fort.35
+&NAMSFC
+  FNGLAC="${FNGLAC}",
+  FNMXIC="${FNMXIC}",
+  FNTSFC="${FNTSFC}",
+  FNSNOC="${FNSNOC}",
+  FNZORC="${FNZORC}",
+  FNALBC="${FNALBC}",
+  FNALBC2="${FNALBC2}",
+  FNAISC="${FNAISC}",
+  FNTG3C="${FNTG3C}",
+  FNVEGC="${FNVEGC}",
+  FNVETC="${FNVETC}",
+  FNSOTC="${FNSOTC}",
+  FNSMCC="${FNSMCC}",
+  FNVMNC="${FNVMNC}",
+  FNVMXC="${FNVMXC}",
+  FNSLPC="${FNSLPC}",
+  FNABSC="${FNABSC}",
+  FNMSKH="${FNMSKH}",
+  FNTSFA="${FNTSFA}",
+  FNACNA="${FNACNA}",
+  FNSNOA="${FNSNOA}",
+  LDEBUG=.false.,
+  FSLPL=${FSLPL},
+  FSOTL=${FSOTL},
+  FVETL=${FVETL},
+  FSMCL(2)=${FSMCL2},
+  FSMCL(3)=${FSMCL2},
+  FSMCL(4)=${FSMCL2},
+  ${CYCLVARS}
+ /
+EOF
+
+cat << EOF > fort.36
+ &NAMCYC
+  idim=${CRES}, jdim=${CRES}, lsoil=${LSOIL},
+  iy=${iy}, im=${im}, id=${id}, ih=${ih}, fh=${FHOUR},
+  deltsfc=${DELTSFC},ialb=${IALB},use_ufo=${use_ufo},donst="${DONST}",
+  do_sfccycle=${DO_SFCCYCLE},do_landincr=${DO_LANDINCR},isot=${ISOT},ivegsrc=${IVEGSRC},
+  zsea1_mm=${zsea1},zsea2_mm=${zsea2},MAX_TASKS=${MAX_TASKS_CY},
+  frac_grid=${FRAC_GRID}
+ /
+EOF
+
+
+cat << EOF > fort.37
+ &NAMSFCD
+  NST_FILE="${NST_FILE}",
+  DO_SOILINCR=${GCYCLE_DO_SOILINCR},
+  DO_SNOWINCR=${GCYCLE_DO_SNOWINCR},
+  INTERP_LANDINCR=${GCYCLE_INTERP_LANDINCR},
+  lsoil_incr=${LSOIL_INCR},
+ /
+EOF
+
+"${APRUNCY}" "${CYCLEXEC}" 1>"${PGMOUT}" 2>"${PGMERR}"
```

Comment:
My bad.  `APRUNCY` should not be double-quoted.

```suggestion
${APRUNCY} "${CYCLEXEC}" 1>"${PGMOUT}" 2>"${PGMERR}"
```